pkg/webhook/resources/setting/validator.go

```diff
@@ -1144,6 +1169,83 @@ func (v *settingValidator) checkStorageNetworkValueVaild() error {
 	return nil
 }
 
+func (v *settingValidator) checkVlanStatusReady(config *storagenetworkctl.Config) error {
+	vsList, err := v.vsCache.List(labels.Set{
+		utils.KeyClusterNetworkLabel: config.ClusterNetwork,
+	}.AsSelector())
+	if err != nil {
+		return err
+	}
+
+	if len(vsList) == 0 {
```

Comment:
seems better to have a `checkClusterNetwork` function to sure the `ClusterNetwork` is existing at first, it will emit an exact error like `the ClusterNetwork xyz is not existing`

then the `checkVlanStatusReady`

Reply:
@w13915984028 Added check for existence of cluster network as well.Thanks